test/test_prototype_transforms.py

```diff
@@ -222,6 +231,58 @@ def test_random_resized_crop(self, transform, input):
         transform(input)
 
 
+@pytest.mark.parametrize(
+    "sample",
+    [
+        [make_pil_image(), make_vanilla_tensor_image(), make_vanilla_tensor_image()],
+        [make_vanilla_tensor_image(), make_pil_image(), make_vanilla_tensor_image()],
+        [make_vanilla_tensor_image(), make_vanilla_tensor_image(), make_pil_image()],
+        [make_image(), make_vanilla_tensor_image(), make_vanilla_tensor_image()],
+        [make_vanilla_tensor_image(), make_image(), make_vanilla_tensor_image()],
+        [make_vanilla_tensor_image(), make_vanilla_tensor_image(), make_image()],
```

Comment:
Did you remove the previous use of `product()` because this is easier to debug?

Reply:
No, I was just blind to the fact that we can also use `repeat=3` to create inputs with two tensor images inside. Although we need to use `permutations` now. This will take us from 9 to 24 tests, but I guess that is still acceptable?